js/ui/lookup.js

```diff
@@ -303,11 +311,9 @@ const Lookup = DropDownList.inherit({
 
                     showPopupTitle: false,
 
-                    position: {
```

Comment:
Should we change the description of [position](https://js.devexpress.com/Documentation/20_1/ApiReference/UI_Widgets/dxLookup/Configuration/#position) for Material theme

Reply:
A technical writer will fix after merge